### DISCUSSION TITLE
What are the convolved data?

### DISCUSSION THREAD

**Discussion by mapfiable** [sep-developers/sep#99] [Q&A] [answered]
In the output array of [`sep.extract`](https://sep.readthedocs.io/en/v1.0.x/api/sep.extract.html) there are many entries with reference to the "convolved data" (e.g. `'cflux'`, `'cpeak'`, etc.). What is this data and how is it calculated?

**Comment by kbarbary** [comment, marked as answer]
The "convolved data" is the input image convolved with the `filter_kernel` array.

**Reply by mapfiable** [reply]
Ooh, I see. I didn't make that connection. I mean, not that it isn't nice to have the `flux` for the filtered image, but aren't they difficult to interpret? Like, what do these values actually mean? As far as I understand, the convolution is only a technical means to enhance the detection results, but doesn't have a physical representation. Like, the convolved values are just "arbitrarily" worped. Can the convolved values tell us something the "real" values cannot? Is there a scenario where I would prefer the convolved values? I would naiively assume that the unconvolved values are always closer to the truth.
